apps/ledger-live-mobile/src/dynamicContent/ContentCardsCategory/Header.tsx

```diff
@@ -19,18 +20,19 @@ const Header = ({ title, description, cta, link }: HeaderProps) => {
   };
 
   return (
-    <Flex mx={6} mb={6}>
+    <Flex mx={6} mb={6} alignItems={centered ? "center" : undefined}>
       {title || (link && cta) ? (
         <Flex flexDirection="row" justifyContent="space-between" alignItems="center">
           <Text
             variant="h5"
             fontWeight="semiBold"
             numberOfLines={1}
-            width={link && cta ? "80%" : "100%"}
+            textAlign={centered ? "center" : "left"}
+            width={link && cta && !centered ? "80%" : "100%"}
           >
             {title}
           </Text>
-          {link && cta ? (
+          {link && cta && !centered ? (
```

Comment:
If centered so no link?

Reply:
Yep asked by growth I guess it's to prevent overlap